airflow-core/src/airflow/ui/src/queries/useClearTaskInstances.ts

```diff
@@ -46,14 +49,49 @@ export const useClearTaskInstances = ({
   const queryClient = useQueryClient();
   const { t: translate } = useTranslation("dags");
 
-  const onError = (error: Error) => {
-    toaster.create({
-      description: error.message,
-      title: translate("dags:runAndTaskActions.clear.error", { type: translate("taskInstance_one") }),
-      type: "error",
-    });
+  const onError = (error: unknown) => {
+    // Narrow the type safely
+    if (typeof error === "object" && error !== null) {
+      const apiError = error as ApiError;
+
+      const detail = typeof apiError.detail === "string" ? apiError.detail : "";
+      const ifDetailIsIncluded =
+        typeof detail === "string" && detail.includes("AirflowClearRunningTaskException");
+
+      if (detail !== "" && ifDetailIsIncluded === true) {
```

Comment:
```suggestion
      if (detail.includes("AirflowClearRunningTaskException")) {
```

I feel like you can simplify this

Reply:
Additionally, only the description is changing. So let's make that a variable and then we only need to call toaster.create() once.